Test1/src/leetcode/LeetCode39.java

```diff
@@ -0,0 +1,48 @@
+package leetcode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * https://leetcode.com/problems/combination-sum/description/
+ * @author jhm95
+ */
+public class LeetCode39 {
+	
+	List<List<Integer>> answerList = new ArrayList<>();
+
+	public List<List<Integer>> combinationSum(int[] candidates, int target) {
+		
+		findAnswer(candidates, new ArrayList<>(), target);
+
+		return answerList;

```

Comment:
It's not a good pattern to return a "global" list.

I think you could just declare an answerList inside this function, not at line 12.